udata_piwik/client.py

```diff
@@ -19,6 +19,8 @@ def analyze(method, **kwargs):
         'idSite': current_app.config['PIWIK_ID'],
         'method': method,
         'format': kwargs.pop('format', 'json'),
+        'token_auth': current_app.config['PIWIK_AUTH']
```

Comment:
Maybe test if `current_app.config['PIWIK_AUTH']` is set and then optionnaly add it.